app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -61,41 +71,34 @@ keywords:
   - Settlement Scheme
 ---
 $applications-open$
+  
+If you are considering coming to England to teach languages or physics, you could be eligible for financial help from the UK government worth £10,000.  
 
+The international relocation payment (IRP) is designed to cover some of the costs of moving to England, including: 
 
+* the costs of visas 
 
-The information on this page is for:
-
-* [non-UK teachers of languages, physics and general or combined science when it includes physics](#criteria-for-teachers)
-* [non-UK trainee teachers of languages and physics](#criteria-for-trainee-teachers-fee-paying-and-salaried)   
-
-If you are considering coming to England to teach or train to teach languages or physics, you could be eligible for financial help from the UK government worth £10,000.  
-
-The international relocation payment (IRP) is designed to cover some of the costs of moving to England, including:
-
-* the costs of visas
-
-* the immigration health surcharge
+* the immigration health surcharge 
 
-* other relocation expenses
+* other relocation expenses 
 
-All eligible applicants will receive a single payment of £10,000 from the Department for Education around the end of their first term training or working in England. The IRP is a grant – you will not have to pay this money back. You can only receive the IRP once.
+The IRP is a grant – you will not have to pay this money back. You can only receive the IRP once.
 
 [Learn more about teaching in England if you trained outside the UK](/non-uk-teachers/teach-in-england-if-you-trained-overseas).
 
-[Learn more about training to teach in England as a non-UK citizen](/non-uk-teachers/train-to-teach-in-england-as-an-international-student).
-
-## Criteria for teachers  
+## Criteria  
```

Comment:
@laurahermionetennant92 As it's a new change I'd keep for teachers in to make it clear, again in case people don't read the box at the top and the same with Teaching subject

Reply:
OK yeah you're probs right